src/Symfony/Bundle/FrameworkBundle/Resources/config/fragment_renderer.xml

```diff
@@ -12,8 +12,8 @@
     <services>
         <service id="fragment.handler" class="Symfony\Component\HttpKernel\DependencyInjection\LazyLoadingFragmentHandler">
             <argument type="service" id="service_container" />
-            <argument>%kernel.debug%</argument>
             <argument type="service" id="request_stack" />
+            <argument>%kernel.debug%</argument>
```

Comment:
The Bc break is that existing code passing the RequestStack already (the recommended usage) would now break